quickstart.js

```diff
@@ -25,16 +22,11 @@ var client, application, account, group = null;
 var accountEmail = unique('jlpicard') + '@mailinator.com';
 
 // ==================================================
-// Step 1 - Load the API Key file and create a Client
+// Step 1 - Create a client and wait for it to ready
```

Comment:
I guess it will break the doc layout. Because it seems like all sentences are kept intentionally short in order to adhere to the `====`.